src/ScribuntoLuaLibrary.php

```diff
@@ -93,4 +106,118 @@ public function getPropertyType( $propertyName = null ) {
 		return array( $property->findPropertyTypeID() );
 	}
 
+	/**
+	 * This mirrors the functionality of the parser function #info and makes it available to lua.
+	 *
+	 * @param string	$text	text to show inside the info popup
+	 * @param string    $icon   identifier for the icon to use
+	 *
+	 * @uses \SMW\ParserFunctionFactory::__construct
+	 *
+	 * @return string[]
+	 */
+	public function info( $text, $icon='info' )
+	{
+		$parser = $this->getEngine()->getParser();
+
+		# do some parameter processing
+		if ( !trim($text) || ! is_string($text) ) {
```

Comment:
extraced two function fragments that coincidenlty can be reused in subobject() as well. integrating them there later